src/TriggerCandidateMakerMichelElectron.cpp

```diff
@@ -34,7 +40,7 @@ TriggerCandidateMakerMichelElectron::operator()(const TriggerActivity& activity,
 
       // add_window_to_record(m_current_window);
       // dump_window_record();
-      // TLOG(1) << "Constructing trivial TC.";
+      TLOG_DEBUG(TLVL_DEBUG_LOW) << "[TCM:ME] Constructing trivial TC.";
```

Comment:
```suggestion
      TLOG_DEBUG(TLVL_DEBUG_LOW) << "[TCM:ME] Constructing a TC.";
```

I know this is not your doing, but might as well correct this... why "Constructing trivial TC"? Why "trivial"? Are there non-trivial TCs? I think just "Constructing a TC" would be fine here, and less confusing...

Reply:
I have not been generally modifying the messages (outside of few examples in MLT where I added more information). I agree with your change.